specifyweb/frontend/js_src/lib/components/Header/index.tsx

```diff
@@ -194,9 +205,21 @@ export function MenuButton({
   readonly props?: TagProps<'a'> & TagProps<'button'>;
 }): JSX.Element | null {
   const [position] = userPreferences.use('header', 'appearance', 'position');
+  const [isSideBarLight] = userPreferences.use(
+    'general',
+    'appearance',
+    'lightSideBarBackground'
+  );
+  const isDarkMode = useDarkMode();
```

Comment:
(optional) create a hook that returns `readonly [isDarkMode: boolean, isMenuDarkMode: boolean]` and use it in two places